src/Redux/Store/Randomizer/BasicNemesisCards/RandomCards/reducerHelpers/__test__/helpers/getRandomBasicNemesisCardsByTier.test.ts

```diff
@@ -197,7 +206,8 @@ describe('getRandomBasicNemesisCardsByTier()', () => {
   it('should get between 1 and 2 minions for four players tier 1', () => {
     const result = getRandomBasicNemesisCardsByTier(cards, 1, 4)
 
-    const minions = result.filter(card => card.type === 'Minion')
+    const regex = new RegExp('minion\\d*', 'g')
+    const minions = result.filter(card => card.id.match(regex))
 
     expect(minions.length).toBeGreaterThanOrEqual(1)
```

Comment:
Reminder:

When you go for your test refactoring, use the seed to generate the cards an make a deep equality check of the  array. You might keep the `toBeLessThan`-check and so on, but don't need to, as they wouldn't really add much, if we have the deep equality check. (but they don't hurt either)

Reply:
@on3iro As I'm refactoring the tests, I just saw that we already have seeded tests in addition to the non seeded tests highlighted above. Should I still add the seed here, too? Or are the seeded tests enough and we don't need the additional checks if the minion amount is in range? My intention here was to also test, that the amount of minions is always in range. We can not achive that result, when we add a seed for the highlighted test, or am I wrong?